src/migration/1699284661116-UserPersonalizedDigestSubscribeTrigger.ts

```diff
@@ -0,0 +1,27 @@
+import { MigrationInterface, QueryRunner } from "typeorm";
+
+export class UserPersonalizedDigestSubscribeTrigger1699284661116 implements MigrationInterface {
+    name = 'UserPersonalizedDigestSubscribeTrigger1699284661116'
+
+    public async up(queryRunner: QueryRunner): Promise<void> {
+      queryRunner.query(`
+        CREATE OR REPLACE FUNCTION user_personalized_digest_subscribe()
+          RETURNS TRIGGER
+          LANGUAGE PLPGSQL
+          AS
+        $$
+        BEGIN
+          INSERT INTO user_personalized_digest ("userId", "preferredTimezone", "preferredHour", "preferredDay") VALUES (NEW.id, COALESCE(NULLIF(NEW.timezone, ''), 'Etc/UTC'), 8, 3);
```

Comment:
We can add `on conflict do nothing` just in case for testing. I see you have to clean the table every time

Reply:
Yeah, it will not help in tests since its a different issue of being created before in fixtures, but good to add I think nevertheless.